handlers/handler-loadbalance/src/main/java/io/servicecomb/loadbalance/CseServer.java

```diff
@@ -44,6 +50,18 @@
    */
   private AtomicInteger continuousFailureCount = new AtomicInteger(0);
 
+  /**
+   * Count the total invocations in the last 10 sec.
+   */
+  private RollingWindowBucketedCounter totalInvocationCount =
```

Comment:
each time instance list changed, will create new CseServer instance
so cseServer should be stateless